docs/14_extending-kedro.md

```diff
@@ -1,135 +1,44 @@
-# Customise Kedro CLI
+# Adding a custom CLI command
```

Comment:
```suggestion
# How to add a custom CLI command
```

Reply:
Maybe rename this documentation file since it's not so much extending Kedro any more but how to add a custom CLI command. Call it something like `custom-cli-commands.md` ??